### PR TITLE
Add stepDefinitionName and stepDefinitionType to flows

### PR DESCRIPTION
**Pull request by akshaysonvane**

### CODE REVIEW

File: marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json [hunk 1]
```diff
@@ -40,8 +40,8 @@
     },
     "3": {
       "language": "zxx",
-      "name": "default-mastering",
-      "type": "MASTERING",
+      "stepDefinitionName": "default-mastering",
+      "stepDefinitionType": "MASTERING",
       "modulePath": "/data-hub/5/builtins/steps/master/default/main.sjs",
```
Comment (hao1st) [inline marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: "modulePath": "/data-hub/5/builtins/steps/mastering/default/main.sjs"

File: marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
   "steps": {
     "1": {
       "language": "zxx",
-      "name": "default-ingestion",
-      "type": "INGESTION",
+      "stepDefinitionName": "default-ingestion",
+      "stepDefinitionType": "INGESTION",
       "modulePath": "/data-hub/5/builtins/steps/ingest/default/main.sjs",
```
Comment (hao1st) [inline marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: "modulePath": "/data-hub/5/builtins/steps/ingestion/default/main.sjs",

Comment (srinathgit) [inline marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: "/data-hub/5/builtins/steps/ingest/default/main.sjs" -> "/data-hub/5/builtins/steps/ingestion/default/main.sjs"

File: marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json (hunk 1 above)
Comment (akshaysonvane) [reply marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: Update https://github.com/marklogic/marklogic-data-hub/pull/2232/commits/87dbfa3e4ad48110c17e9ba9fde86dd424973db2

File: marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json (hunk 2 above)
Comment (akshaysonvane) [reply marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: https://github.com/marklogic/marklogic-data-hub/pull/2232/commits/87dbfa3e4ad48110c17e9ba9fde86dd424973db2

Comment (akshaysonvane) [reply marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: Updated https://github.com/marklogic/marklogic-data-hub/pull/2232/commits/87dbfa3e4ad48110c17e9ba9fde86dd424973db2

File: marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json (hunk 1 above)
Comment (akshaysonvane) [reply marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: Updated https://github.com/marklogic/marklogic-data-hub/pull/2232/commits/87dbfa3e4ad48110c17e9ba9fde86dd424973db2

File: marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json (hunk 2 above)
Comment (akshaysonvane) [reply marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: Updated https://github.com/marklogic/marklogic-data-hub/pull/2232/commits/87dbfa3e4ad48110c17e9ba9fde86dd424973db2

Comment (akshaysonvane) [reply marklogic-data-hub/src/main/resources/scaffolding/defaultFlow.flow.json]: Updated https://github.com/marklogic/marklogic-data-hub/pull/2232/commits/87dbfa3e4ad48110c17e9ba9fde86dd424973db2

**Pull request merged**